include/couch_mrview.hrl

```diff
@@ -98,7 +98,8 @@
     should_close = false,
     buffer = [],
     bufsize = 0,
-    threshold = 1490
+    threshold = 1490,
+    sent = false
```

Comment:
May be name it `metadata_sent`? Because it's confusing flag. First chunk sent? Whole response sent? Any data sent?

Or, since feature is `sorted=true|false`, may be have the similar field there instead?